### PR TITLE
Kotlin 1.8.20 & Dokka 1.8.10

### PR DESCRIPTION
**Pull request by Goooler**
## Description

https://github.com/JetBrains/kotlin/releases/tag/v1.8.20
https://github.com/Kotlin/dokka/releases/tag/v1.8.10

## Checklist

- [x] PR description added
- [ ] tests are added
- [ ] KtLint has been applied on source code itself and violations are fixed
- [ ] [documentation](https://pinterest.github.io/ktlint/) is updated
- [ ] `CHANGELOG.md` is updated

In case of adding a new rule:
- [ ] Rule is added to [rules documentation](https://pinterest.github.io/ktlint/rules/standard/)

### CODE REVIEW

**Comment by paul-dingemans** [comment]
The reference to the Kotlin release (1.8.20) does not or no longer exist. I found 1.8.20-RC2 (https://github.com/JetBrains/kotlin/releases/tag/v1.8.20-RC2) as latest version. Should I still proceed with merge?

**Comment by Goooler** [comment]
It was published to MavenCentral hours ago, but hasn't been released on GitHub, we can just wait a moment.

**Comment by paul-dingemans** [comment]
> It was published to MavenCentral hours ago, but hasn't been released on GitHub, we can just wait a moment.

That at least explains why the build succeeded. I was confused about a build succeeding with an unreleased version ;-)

**Comment by Goooler** [comment]
They always delay the releases on GitHub 🤣

**Comment by Goooler** [comment]
It's out.

**Comment by paul-dingemans** [comment]
> It's out.

Build broke on cache not having enough space. I have restarted the failed jobs. We have to monitor whether this happens more often.

File: .github/workflows/gradle-pr-build.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ jobs:
         # When changing the list of JDK versions, the build configuration has to be changed by a repository admin. See
         # https://github.com/pinterest/ktlint/pull/1787#issuecomment-1409074092
         jdk: [ 8, 11, 17, 19 ]
+        exclude: # windows with JDK8 are *really* flaky
+          - os: windows-latest
+            jdk: 8
```
Comment (Goooler) [inline .github/workflows/gradle-pr-build.yml:26]: Disabled Java 8 tests on Windows like [Detekt did](https://github.com/detekt/detekt/blob/43f0796c54a179bfd0e0e94c7c79128771bbbf2a/.github/workflows/pre-merge.yaml#L25-L27), if so we need to change the status check in repo settings.

Comment (paul-dingemans) [reply .github/workflows/gradle-pr-build.yml:26]: @shashachu Can you reconfigure as described in https://github.com/pinterest/ktlint/pull/1787#issuecomment-1409074092?

Comment (shashachu) [reply .github/workflows/gradle-pr-build.yml:26]: done. removed windows jdk 8

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [versions]
-kotlin = "1.8.10"
-kotlinDev = "1.8.20-Beta"
+kotlin = "1.8.20"
+kotlinDev = "1.8.20"
```
Comment (shashachu) [inline gradle/libs.versions.toml:3]: is there no 1.8.30 dev version yet?

**Review by shashachu — APPROVED** [review]

File: gradle/libs.versions.toml (hunk 2 above)
Comment (Goooler) [reply gradle/libs.versions.toml:3]: No public released version yet.

**Pull request merged**